core/store/src/main/java/uk/gov/gchq/gaffer/store/library/FileGraphLibrary.java

```diff
@@ -122,12 +122,12 @@ protected void _addSchema(final String schemaId,
     protected void _addProperties(final String propertiesId,
                                   final StoreProperties properties) {
         if (null != properties) {
-            getPropertiesPath(propertiesId).toFile().getParentFile().mkdirs();
+            Boolean dirCreated = getPropertiesPath(propertiesId).toFile().getParentFile().mkdirs();
             try (FileOutputStream propertiesFileOutputStream = new FileOutputStream(getPropertiesPath(propertiesId).toFile())) {
                 properties.getProperties().store(propertiesFileOutputStream, null);
             } catch (final IOException e) {
                 throw new IllegalArgumentException("Could not write " +
-                        "properties to path: " + getPropertiesPath(propertiesId), e);
+                        "properties to path: " + getPropertiesPath(propertiesId) + ". Directory created: " + dirCreated, e);
```

Comment:
Could use `String.format` instead...

Reply:
That does look more readable here, I've made the change.